tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -412,3 +412,59 @@ def test_duplicate_autolog_second_overrides(duplicate_autolog_tf_estimator_run):
     client = mlflow.tracking.MlflowClient()
     metrics = client.get_metric_history(duplicate_autolog_tf_estimator_run.info.run_id, 'loss')
     assert all((x.step - 1) % 4 == 0 for x in metrics)
+
+
+@pytest.mark.large
```

Comment:
Nit: I would move these tests a bit up in the file with the other `tf.keras` tests since the tests are split by `tf.estimator` and `tf.keras` models.